.github/workflows/cd-prod.yml

```
@@ -50,62 +49,48 @@ jobs:
+            TO_DELETE=$(echo $IMAGES | jq -c ".[:$((COUNT-5))] | [{imageDigest: .[]}]")
+            if [ "$TO_DELETE" != "[]" ]; then
+              aws ecr-public batch-delete-image \
+                --region us-east-1 \
+                --repository-name $ECR_APP_NAME \
+                --image-ids "$TO_DELETE"
+            else
+              echo "No images to delete."
+            fi
+          else
+            echo "Less than or equal to 5 images, skipping prune."
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)